### ISSUE TITLE
When collapsing any tree item VoiceOver is not announcing the collapsed state.:A11y_Visual Studio Code_Github_Copilot_Screen reade

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vscode#286751]
Try [ES Chat](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Feng.ms%2Fdocs%2Fcoreai%2Fdevdiv%2Fone-engineering-system-1es%2F1es-docs%2Fes-chat%2Findex&data=05%7C02%7Cv-pawanksing%40microsoft.com%7C26cff27270c448f48a0e08de3f569146%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639017838221175207%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=ju8Raw6mIhT5G3gWbZSkS0CAN2zkv2ycA5ifSSqoxOw%3D&reserved=0) to learn more about the MAS rule and how to fix the issue. If you need more help, use our [Teams channel](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fteams.microsoft.com%2Fl%2Fchannel%2F19%253ADeCtYeREyQHw5jYsz9PPc3hpVw48m-tOlTkQ1jEn0eM1%2540thread.tacv2%2FCAI%2520Accessibility%2520Support%2520Channel%3FgroupId%3Df5eb428c-9e6e-4c16-8143-33fe3961a9ae%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=05%7C02%7Cv-pawanksing%40microsoft.com%7C26cff27270c448f48a0e08de3f569146%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639017838221198042%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=DCNVsZe8oDZWqm97AI0ee4OoWWecR6zw6qlhTuzP7MI%3D&reserved=0) or [office hours](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Foutlook.office.com%2Fbook%2Fmicrosoft2779007%2540microsoft.onmicrosoft.com%2Fs%2FbLeGBltMW0qI0WRLsTqx2A2%3Fismsaljsauthenabled&data=05%7C02%7Cv-pawanksing%40microsoft.com%7C26cff27270c448f48a0e08de3f569146%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639017838221214666%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=CnCCdDgEmll2oUQUdMjXusAgV8PneqZVYvzgxAfk7Ik%3D&reserved=0).
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## GitHub Tags
#A11yMAS; #A11yTCS; #MAC; #A11ySev3; #DesktopApp; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Jan2024; #WCAG4.1.2; #FTP; #Screen Reader; #Device; #ES Chat;

## Environment Details:
Application: VS Code
Visual studio code version: 1.108.0
MAC M2

## Repro Steps
1. Open Vs code
2. GitHub copilot app modernization
3. TAB till Tree item  
4. observe the issue

## Actual Experience.
When collapsing any tree item VoiceOver is not announcing the collapsed state.

## Expected Experience
When collapsing any tree item VoiceOver should announce the collapsed state.

## User Impact:
Screen reader user will not sure about the completion of action.

## Attachment

https://github.com/user-attachments/assets/b798d847-ccff-46fa-bbd5-57979a73e52d
